kubens

```diff
@@ -118,6 +118,8 @@ choose_namespace_interactive() {
 set_namespace() {
   local ctx prev
   ctx="$(current_context)" || exit_err "error getting current context"
+  [[ -z "$ctx" ]] && exit_err "current context is empty"
```

Comment:
I can't imagine a case where this returns empty string.  Can you clarify? Ideally the previous line should fail.

Reply:
this was also not necessary